src/clusterfuzz/_internal/swarming/__init__.py

```diff
@@ -118,6 +121,4 @@ def push_swarming_task(command, download_url, job_type):
   }
   swarming_server = _get_swarming_config().get('swarming_server')
   url = f'https://{swarming_server}/prpc/swarming.v2.Tasks/NewTask'
-  r = requests.post(
-      url=url, data=task_spec.SerializeToString(), headers=headers)
-  logs.log(r.status_code)
+  utils.post_url(url=url, data=task_spec.SerializeToString(), headers=headers)
```

Comment:
I'm surprised this works. I would have expected the HTTP endpoint to ingest JSON, but we're passing a binary-serialized protobuf?

Reply:
Thanks for catching that, I had made a mistake here.